assets/scss/education.scss

```diff
@@ -0,0 +1,314 @@
+@import "./style.css";
+/* Dark and Light theme */
+/*color of education page heading*/
+/* -----Education Section CSS----- */
```

Comment:
These comments are totally misplaced and makes the project very hard to read now